ext/reflection/tests/mixed_type.phpt

```diff
@@ -17,8 +17,17 @@ $method = new ReflectionMethod($a, "test");
 var_dump($object->getProperty("a")->getType()->getName());
 var_dump($method->getParameters()[0]->getType()->getName());
 var_dump($method->getReturnType()->getName());
+
+var_dump((string) $object->getProperty("a"));
```

Comment:
```suggestion
var_dump((string) $object->getProperty("a")->getType());
```
etc

Reply:
Ohh, I accidentally removed the `->getType()` part